lib/rake/circle.rake

```diff
@@ -163,19 +163,6 @@ namespace :circle do
       RakeUtils.rake_stream_output 'seed:cached_ui_test'
     end
   end
-
-  desc 'Rebuild dashboard assets with updated locals.yml settings before running UI tests'
-  task :recompile_assets do
-    if CircleUtils.tagged?(SKIP_UI_TESTS_TAG)
-      ChatClient.log "Commit message: '#{CircleUtils.circle_commit_message}' contains [#{SKIP_UI_TESTS_TAG}], skipping UI tests for this run."
-      next
-    end
-
-    system 'rm', '-rf', dashboard_dir('tmp', 'cache', 'assets')
```

Comment:
It looks like this line was intended to clear dashboard assets between CircleCI test runs, although it's hard to know whether it was effective. Are files in dashboard/public/assets shared between drone test runs via any caching mechanism? If so, I could add a a step to remove all existing assets before `rake build` in `ui_tests.sh`.

Reply:
Nothing is saved between runs currently 👍